### ISSUE TITLE
wooNuxtSEO is not a function

### ISSUE THREAD

**Issue by tzhf** [scottyzen/woonuxt#218]
Hi there,
I'm running a fresh install using Local and finally got the SSL working locally, i'm able to login/manage my customer account, and load all products.
However when i click on a single product nothing happens and i'm getting this in the console :

![image](https://github.com/user-attachments/assets/17ec42b1-dbb9-4970-a342-9b1085d3b340)

And on the product/slug route :

![image](https://github.com/user-attachments/assets/fdaf5a31-ce71-4f3c-8c8e-ee95e2d6e671)

I looked at the doc but found nothing about this function or any mention in the issues.

Cheers

**Comment by grandant** [comment]
TL;DR: **You must have at least one product with a price set** in the backend (All Products menu) to avoid WooNuxt falling back to default settings.

This happened to me seemingly out of nowhere while everything was working fine. I removed the last existing price on my products while testing, which must have triggered the issue for me. It was really frustrating because I reverted every single change to the WooNuxt template and finally tried a clean install before realizing that the root cause must be connected with the backend.

Without at least one product price, the **WooNuxt Settings plugin** falls back to its defaults, which impacts **SEO metadata**, the **primary color** settings, etc.

Hopefully, this helps anyone in the same position.
@scottyzen Can we consider this to be expected behavior, or is it a bug?
